src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1281,6 +1281,14 @@ public void OnRigidityChanged(int desiredRigidity)
             return;
         }
 
+        // Make sure that if there are no mutation points the player cannot drag the slider
+        // when the cost is rounded to zero
+        if (Editor.MutationPoints == 0 && cost == 0)
```

Comment:
I noticed that when playing with the mutation cost of 1.9, dragging the rigidity slider might bring the MP down to -1. So, I think that this check should be updated to be able to handle any mutation cost. Something like `Editor.MutationPoints < cost + 1 && CostMultiplier % 1 != 0` should probably work (the second part checking if the number is non-integer and might have rounding issues).